help/access/file-shares.md

```diff
@@ -1,302 +1,237 @@
 
-> Key concept: _file share_
->
-> We use the term _file share_ when referring to the following:
->
->* Connections to folders and files located on a Datagrok server that you access with a Datagrok client.
->* Any folder or file in the **File Manager** that has been shared with others.
->
->_File shares_ are entities<!--inseert a link-->, meaning you can perform a standard set of operations against them (for
-> example,
-> annotate, set access privileges, [use in automation workflows](data-pipeline.md), or enable discovery by other
-> Datagrok
-> users).
->
-> Note: For enterprise users, the Datagrok administrator defines which file shares you can access and which privileges
-> you have in them.
+:::
 
-## Getting data
+:::note developers
 
-Each Datagrok user has a personal directory for files and folders they want to upload directly to Datagrok. This
-personal directory is accessible in the **Folder Tree** under the name _Home_. The **Home** directory is created
-automatically upon signup. By default, files and folders in your **Home** directory are visible to you only.
+You can [create custom connectors](create-custom-connectors.md) and [read files programmatically](../develop/how-to/access-data.md/#reading-files).
 
-In addition, you can connect network folders hosted on supported data sources, and access files and directories shared
-with you.
+:::
 
-### Supported data sources
+## Connecting to file storage
 
-You can connect to files and folders located on a remote server or a mapped drive. Out-of-the-box, Datagrok provides
-connectors to the following data sources:
+### Adding connection
 
-* [An Amazon Simple Storage Service (S3) bucket](connectors/s3.md)
+![File share connection parameters](add-a-file-share.gif)
 
-* [Windows and Linux network shares](connectors/files.md)
+To connect to your file storage, follow these steps:
 
-  > Note: Connecting to an SMB file share (that is, a file share mounted on a server, such as Linux or Windows network
-  > shares) is only available for on-premise deployment and is not available on the public Datagrok instance
-  > (public.datagrok.ai).
+1. Go to **Data** > **Files**.
+1. Open the **New file share** dialog (**Toolbox** > **Actions** > **New file share**). Alternatively, click the **New file share** icon on the **Menu Riboon**.
+1. In the the **New file share** dialog, choose the data source from the **Data Source** dropdown. The dialog updates with connection-specific parameters.
+1. Set the parameters.
+1. Click **TEST** to test the connection, then click **OK** to save it. A pop-up notification displays the connection status.
 
-* [Dropbox](connectors/dropbox.md)
+Some connection parameters have unique characteristics, and it's important to specify them correctly:
 
-* [Git](connectors/git.md)
+* _Directory path_. When connecting to the root directory, leave the **Dir** field empty. Otherwise, enter a directory path.
```

Comment:
_Directory path_. When connecting to the root directory, leave the **Dir** field empty. Otherwise, enter a directory path.

Too many words. Should be "relative path to the directory that you want to expose"

Reply:
It's not the same though. You dropped the first bit, which contains improtant info